cfgov/jinja2/v1/adult-financial-education/index.html

```diff
@@ -0,0 +1,533 @@
+                </a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link icon-link icon-link__download"
+                   href="https://files.consumerfinance.gov/f/documents/201606_cfpb_consumer-voices-on-automobile-financing.pdf">
+                <span class="a-link_text">Consumer voices on automobile financing</span>
+                </a>
+            </li>
+        </ul>
+    </section>
+    <section class="block resource-section">
+        <div class="resource-section_header">
+            <h2 id="tools-for-financial-educators">
+                Tools for financial educators
+            </h2>
+            <p>Review a few tools and techniques to improve your effectiveness and service to consumers.</p>
+        </div>
+        <ul class="m-list
+                   m-list__links
+                   m-list__two-col">
+            <li class="m-list_item">
+                <a class="m-list_link a-link a-link__icon cf-icon cf-icon__after cf-icon-download" href="https://www.consumerfinance.gov/f/documents/201606_cfpb_take-control-of-your-auto-loan-guide.pdf"><span class="a-link_text">Take control of your auto loan: A step-by-step guide</span> </a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link" href="/blog/the-newcomers-guides-to-managing-money/">Newcomers Guides to Money Management</a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link" href="/know-before-you-owe/">Know Before You Owe | Mortgages</a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link a-link a-link__icon cf-icon cf-icon__after cf-icon-download" href="https://www.consumerfinance.gov/f/201510_cfpb_spanish-style-guide-glossary.pdf"><span class="a-link_text">Glossary of English-Spanish Financial Terms </span> </a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link a-link a-link__icon cf-icon cf-icon__after cf-icon-download" href="https://www.consumerfinance.gov/f/201510_cfpb_chinese-style-guide-glossary.pdf"><span class="a-link_text">Glossary of English-Chinese Financial Terms</span> </a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link a-link a-link__icon cf-icon cf-icon__after cf-icon-download" href="https://www.consumerfinance.gov/f/201603_cfpb_rules-to-live-by_consumer-voices-report.pdf"><span class="a-link_text">Consumer Voices on Financial Rules to Live By</span> </a>
+            </li>
+            <li class="m-list_item">
+                <a class="m-list_link a-link a-link__icon cf-icon cf-icon__after cf-icon-download" href="https://www.consumerfinance.gov/f/201603_cfpb_rules-to-live-by_ideas-for-financial-educators.pdf"><span class="a-link_text">Financial rules to live by: Ideas for financial educators </span> </a>
```

Comment:
Trivial (and this may be an issue carried over from the previous HTML): If you ensure that there is no space between the text and the closing `</span>` tag, the underline will not extend under the space between the text and the icon. (Applies several places in this PR.)